src/string/wcsncpy.c

```diff
@@ -3,8 +3,13 @@
 wchar_t *wcsncpy(wchar_t *restrict d : itype(restrict _Nt_array_ptr<wchar_t>) count(n), const wchar_t *restrict s, size_t n)
   : itype(_Nt_array_ptr<wchar_t>) count(n)
 {
+
+// n and d are used in the declared bounds of d. But both of them are modified in the loop and hence the declared bounds of d no
```

Comment:
Please indent the comment with the code in this function. Also make sure each line of the comment is no longer than 80 chars.